modules/components/src/AdvancedFacetView/index.js

```diff
@@ -13,12 +13,36 @@ import Input from '../Input';
 import SearchBox from './SearchBox';
 import { filterOutNonValue, injectExtensionToElasticMapping } from './utils.js';
 
+const orderDisplayTreeData = displayTreeData => {
+  const setWithChildrenOrdered = displayTreeData.map(
+    ({ children, ...rest }) => ({
+      ...rest,
+      ...(children
+        ? {
+            children: orderDisplayTreeData(children),
+          }
+        : {}),
+    }),
+  );
+  return [
+    ...orderBy(
+      setWithChildrenOrdered.filter(({ children }) => !children),
```

Comment:
You don't need setWithChildrenOrdered the children here, because there are none 
That also means you can pre filter displayTreeData and remove the check for children in the map.